apis/pkg/v1beta1/function_types.go

```diff
@@ -34,7 +34,15 @@ import (
 // +genclient
 // +genclient:nonNamespaced
 
-// Function is the CRD type for a request to deploy a long-running Function.
+// A Function kind represents a
+// [Composition Function](https://docs.crossplane.io/latest/concepts/composition-functions/)
+// in Crossplane.
```

Comment:
While Providers, Configurations, and Functions are all similar they all have different documentation phrasing:

* A Configuration installs OCI "Crossplane package" images into Crossplane.
* A Provider installs the Provider package
* A Function kind represents a Composition Function in Crossplane.

I'd prefer to be consistent. How about:

* A Configuration installs an OCI Crossplane package that extends Crossplane with support for new kinds of XRDs and Compositions.
* A Provider installs an OCI Crossplane package that extends Crossplane with support for new kinds of MR.
* A Function installs an OCI Crossplane package that extends Crossplenw with support for a new kind of composition function.